LICENSE

```diff
@@ -0,0 +1,70 @@
+Copyright (c) 2005-2020 remains with the authors.
+Agda 2 was originally written by Ulf Norell,
+partially based on code from Agda 1 by Catarina Coquand and Makoto Takeyama,
+and from Agdalight by Ulf Norell and Andreas Abel.
+
+Agda 2 is currently actively developed mainly by Andreas Abel,
+Guillaume Allais, Jesper Cockx, Nils Anders Danielsson, Philipp
+Hausmann, Fredrik Nordvall Forsberg, Ulf Norell, Víctor López Juan,
+Andrés Sicard-Ramírez, and Andrea Vezzosi.
+
+Further, Agda 2 has received contributions by, amongst others, Stevan
+Andjelkovic, Marcin Benke, Jean-Philippe Bernardy, Guillaume Brunerie,
+James Chapman, Dominique Devriese, Péter Diviánszky, Olle Fredriksson,
+Adam Gundry, Daniel Gustafsson, Kuen-Bang Hou (favonia), Patrik
+Jansson, Alan Jeffrey, Wolfram Kahl, Wen Kokke, Fredrik Lindblad,
+Francesco Mazzoli, Stefan Monnier, Darin Morrison, Guilhem Moulin,
+Nicolas Pouillard, Nobuo Yamashita, Christian Sattler, Makoto
+Takeyama and Tesla Ice Zhang.  The full list of contributors is
+available at https://github.com/agda/agda/graphs/contributors
+
+Permission is hereby granted, free of charge, to any person obtaining
+a copy of this software and associated documentation files (the
+"Software"), to deal in the Software without restriction, including
+without limitation the rights to use, copy, modify, merge, publish,
+distribute, sublicense, and/or sell copies of the Software, and to
+permit persons to whom the Software is furnished to do so, subject to
+the following conditions:
+
+The above copyright notice and this permission notice shall be
+included in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
+CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+--------------------------------------------------------------------------------
+
+The file src/full/Agda/Utils/Maybe/Strict.hs (and the following
```

Comment:
The LICENSE file is still here.  Shouldn't it be replaced by a new file reflecting the public domain status?
See the text at https://unlicense.org/

Reply:
You're right. I overlooked it. `UNLICENSE` is added.